src/plugin/markers/index.js

```diff
@@ -246,29 +266,74 @@ export default class MarkersPlugin {
 
         labelDiv.addEventListener("click", e => {
             e.stopPropagation();
-            this.wavesurfer.setCurrentTime(time);
+            // Click event is catch if the marker-drop was dispatched.
```

Comment:
`is catch` -> `is caught when the marker-drop event was dispatched.`

Reply:
Fixed ! I’m not fluent in English, feel free to comment if you don’t understand some comment.